impl/sql/mutationstorage/mutations.go

```diff
@@ -75,7 +75,12 @@ func (m *mutations) ReadPage(ctx context.Context, mapID, start, end int64, pageS
 		return 0, nil, err
 	}
 	defer rows.Close()
-	return readRows(rows)
+	max, entryupdates, err := readRows(rows)
+	entries := make([]*pb.Entry, 0, len(entryupdates))
+	for _, e := range entryupdates {
+		entries = append(entries, e.Mutation)
```

Comment:
if you rename the field `Mutation.Mutation` to `Mutation.Entry` then this line of code will read very naturally.